source/index-management/tests/createSearchIndex.yml

```
@@ -0,0 +1,62 @@
+description: "createSearchIndex convenience helper"
```

```suggestion
description: "createSearchIndex"
```

This isn't enforced, but most unified spec tests keep the base filename and top-level description consistent.